chroma-common/Makefile

```diff
@@ -0,0 +1,40 @@
+include ../include/Makefile.version
```

Comment:
Since this is being extracted out, we should also move it out into it's own package.